tweet_recommendations/other_methods/dbscan_based_method.py

```diff
@@ -0,0 +1,94 @@
+from collections import namedtuple
+from itertools import chain
+from typing import *
+
+import numpy as np
+import pandas as pd
+from scipy.spatial.distance import cosine
+from sklearn.cluster import DBSCAN
+from sklearn.neighbors.kd_tree import KDTree
+
+from ..estimator import Estimator
```

Comment:
Jaką przyjęliśmy konwencję nt. względnych/bezwzględnych importów?

Reply:
Osobiście preferuję bezwzględne, chyba że import dotyczy aktualnego katalogu (pojedyncza kropka)

Reply:
Ok, czyli tutaj będzie tak, jak jest w `tweet_recommendations/embeddings/models.py:5`

Reply:
Osobiście optowałbym za używaniem bezwzględnych, chyba że import dotyczy pliku (nie całego modułu) z poziomu tego pliku

Reply:
Z tego co kojarzę, to PEP8 raczej odradza stosowanie względnych importów. Daje jakieś tam pojedyncze wyjątki kiedy jak już bardzo się chce to można, ale lepiej jednak unikać. Używanie tego później jako paczki robi się problematyczne